### ISSUE TITLE
install-deps-libc fails because osname is null for Linux

### ISSUE THREAD

**Issue by thomasmodeneis** [skycoin/skycoin#1586]
```
make install-deps-libc
mkdir -p dist/usr/tmp dist/usr/lib dist/usr/include
mkdir -p dist/libskycoin bin include
wget -O dist/usr/tmp/criterion-v2.3.2--x86_64.tar.bz2 https://github.com/Snaipe/Criterion/releases/download/v2.3.2/criterion-v2.3.2--x86_64.tar.bz2
--2018-05-28 19:23:51--  https://github.com/Snaipe/Criterion/releases/download/v2.3.2/criterion-v2.3.2--x86_64.tar.bz2
Resolving github.com (github.com)... 192.30.253.112, 192.30.253.113
Connecting to github.com (github.com)|192.30.253.112|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2018-05-28 19:23:52 ERROR 404: Not Found.

Makefile:193: recipe for target 'install-deps-libc' failed
make: *** [install-deps-libc] Error 8
```

I will submit a small fix asap.

**Comment by olemis** [comment]
For the record , those instructions are temporary and needed until #1225 will be merged with `develop`.

**Comment by thomasmodeneis** [comment]
Yep that works, as its not using the broken OSNAME condition.
But it directly clones the Criterion on master.
```
install-deps-libc: configure-build ## Install locally dependencies for testing libskycoin
	git clone --recursive https://github.com/Snaipe/Criterion $(BUILD_DIR)/usr/tmp/Criterion
	mkdir $(BUILD_DIR)/usr/tmp/Criterion/build
	cd    $(BUILD_DIR)/usr/tmp/Criterion/build && cmake .. && cmake --build .
	mv    $(BUILD_DIR)/usr/tmp/Criterion/build/libcriterion.* $(BUILD_DIR)/usr/lib/
	cp -R $(BUILD_DIR)/usr/tmp/Criterion/include/* $(BUILD_DIR)/usr/include/
```

This wont lock to a particular version, thus allowing potential problems if master is updated and becomes incompatible.

**Comment by olemis** [comment]
You are definitely right . We should be using https://github.com/skycoin/Criterion as we do it for the [skycoin/skycoin-dev Docker image](https://github.com/skycoin/skycoin/pull/1183/files#diff-97bc474090003351a8382af33079f730R20) .

Thanks for spotting this .

**Comment by olemis** [comment]
@thomasmodeneis should be fixed in fd134aa

**Comment by thomasmodeneis** [comment]
Great, glad to help! :)
Cheers.
